### DISCUSSION TITLE
Maps in the simple-starter repo are not working

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1955] [Q&A] [answered]
For an future project I try to set up the  simple-starter repo.

Repo: https://github.com/christoph-LSN/MI

geojson: https://github.com/christoph-LSN/MI/blob/develop/laender.geojson

_config.yml: https://github.com/christoph-LSN/MI/blob/5d83402dbc3cc308a38381c7f378c99621b3afbd/_config.yml#L35

config_data.yml: https://github.com/christoph-LSN/MI/blob/5d83402dbc3cc308a38381c7f378c99621b3afbd/config_data.yml#L115

I have filled in the reference to my geojson in the config_data.yml and added -map_laxers and map_options in my _config.yml

Looking at the map in indicator 1-1-1: https://christoph-lsn.github.io/MI/1-1-1/
I only can see aa turning wheel. Inspecting the site the console message is: 

Map disabled - please add "map_layers" in site configuration.

I do not know what might be wrong....

**Comment by brockfanning** [comment, marked as answer]
@christoph-LSN The config_data.yml would be considered the "data configuration". The "site configuration" is in the www/_data folder. You would need to do the map layers configuration there: https://github.com/christoph-LSN/MI/blob/develop/www/_data/site_config.yml#L146

Here's more info on the site configuration map_layers setting: https://open-sdg.readthedocs.io/en/latest/maps/#site-configuration-map_options-and-map_layers

A more general issue is that you're putting site configuration in the _config.yml file. Instead you should be putting site configuration in the www/_data/site_config.yml file.

**Comment by christoph-LSN** [comment]
@brockfanning Thank you ! I will try this. 

I haven't noticed that there is the site config file in the www/_data folder.
